client/my-sites/stats/stats-notices/gdpr-cookie-consent-notice.tsx

```diff
@@ -0,0 +1,73 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import NoticeBanner from '@automattic/components/src/notice-banner';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import useNoticeVisibilityMutation from 'calypso/my-sites/stats/hooks/use-notice-visibility-mutation';
+import { useSelector } from 'calypso/state';
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+import { StatsNoticeProps } from './types';
+
+const GDPRCookieConsentNotice = ( { siteId, isOdysseyStats }: StatsNoticeProps ) => {
+	const translate = useTranslate();
+	const adminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
+	const [ noticeDismissed, setNoticeDismissed ] = useState( false );
+
+	const { mutateAsync: postponeNoticeAsync } = useNoticeVisibilityMutation(
+		siteId,
+		'gdpr_cookie_consent',
+		'postponed',
+		7 * 24 * 3600
```

Comment:
As we talked, let's set the postponed time to 2 months.

Reply:
Addressed in [0e27703](https://github.com/Automattic/wp-calypso/pull/91762/commits/0e27703fbcfeb658453a883730a1fdaea7edcde4).